dbt-bigquery/src/dbt/adapters/bigquery/credentials.py

```diff
@@ -71,6 +77,16 @@ class BigQueryCredentials(Credentials):
     client_secret: Optional[str] = None
     token_uri: Optional[str] = None
 
+    # wif
+    audience: Optional[str] = None
```

Comment:
Like we were saying earlier, this should be more descriptive to distinguish it from an aud claim in a JWT I think